content/en/getting_started/application_security/_index.md

```diff
@@ -49,15 +49,18 @@ These security insights are detected from data reported by APM. The insights hel
 
 ### Enable ASM with in-app instructions
 
-Go to the [ASM Setup page][1] and follow the instructions to get started. This includes:
+On the [ASM landing page,][18] follow the instructions to get started. This includes:
 - Guided selection of services that would benefit from ASM.
 - Configuring your Datadog tracing libraries with an environment variable.
 - Restarting your services. </br>
 
-1. [Go to ASM][18] and click **Get Started with ASM**.
-2. Select services exposed to risk that are recommended by Datadog.
+1. Click **Get Started with ASM**.
+2. Select **Get Started** to detect vulnerabilities in open-source libraries (Software Composition Analysis), find and fix code-level vulnerabilities (Code Security), and select Threat Management to find and enable threat detection on your services that are recommended by Datadog.
```

Comment:
```suggestion
2. Select **Get Started** to detect vulnerabilities in open-source libraries (Software Composition Analysis), find and fix code-level vulnerabilities (Code Security), and find and enable threat detection on your services (Threat Management).
```
This sentence was a little hard to parse, since the last item didn't have the same structure as the first two. If the "recommended by Datadog" piece is really key, you might ignore this suggestion, but I think the recommendation aspect is implied by the phrase "threat detection."

Reply:
That does sound much better thank you !